### DISCUSSION TITLE
I wrongly delete kubeconfig created by sealos kubenetes, now I can`t connect to the kubernetes,how can I regenerate the kubeconfig file again without redeploying it again?

### DISCUSSION THREAD

**Discussion by mingmingshiliyu** [labring/sealos#5163] [General] [answered]
I wrongly delete kubeconfig created by sealos kubenetes in ~/.kube/config , now I can`t connect to the kubernetes,how can I regenerate the kubeconfig file again without redeploying it again?

**Comment by bxy4543** [comment]
Is it multiple nodes? you can copy it to other nodes.

**Reply by mingmingshiliyu** [reply]
single node

**Comment by mingmingshiliyu** [comment, marked as answer]
解决了,/etc/kubernetes中有自动备份
